### ISSUE TITLE
Paypal Credit Button shows in minicart and cart page

### ISSUE THREAD

**Issue by MichaelThessel** [magento/magento2#25078]
### Preconditions (*)

1. 2.3.3 all modules up to date

### Steps to reproduce (*)

1. Enable Paypal Expres Checkout
2. Make sure PP Credit is disabled
3. Add product to cart
4. Check mini cart or cart page

### Expected result (*)

1. No additional Paypal Credit Button shows

### Actual result (*)

1. I see an additional PP credit button

![20191015_161602](https://user-images.githubusercontent.com/2926266/66876576-1d138800-ef67-11e9-893b-ab5dceb48d24.png)

**Config**

![20191015_161527](https://user-images.githubusercontent.com/2926266/66876551-0705c780-ef67-11e9-9dc8-c98857d7acb1.png)

```
mysql> select * from core_config_data where path like '%bml%';
+-----------+---------+----------+--------------------------------------------------+---------+---------------------+
| config_id | scope   | scope_id | path                                             | value   | updated_at          |
+-----------+---------+----------+--------------------------------------------------+---------+---------------------+
|      4972 | default |        0 | payment/paypal_express_bml/active                | 0       | 2019-10-08 16:10:43 |
|      4973 | default |        0 | payment/paypal_express_bml/publisher_id          | NULL    | 2019-10-08 16:10:43 |
|      4974 | default |        0 | payment/paypal_express_bml/homepage_display      | 0       | 2019-10-15 16:09:25 |
|      4975 | default |        0 | payment/paypal_express_bml/homepage_position     | 0       | 2019-10-08 16:10:43 |
|      4976 | default |        0 | payment/paypal_express_bml/homepage_size         | 190x100 | 2019-10-08 16:10:43 |
|      4977 | default |        0 | payment/paypal_express_bml/categorypage_display  | 0       | 2019-10-15 16:09:25 |
|      4978 | default |        0 | payment/paypal_express_bml/categorypage_position | 0       | 2019-10-08 16:10:43 |
|      4979 | default |        0 | payment/paypal_express_bml/categorypage_size     | 190x100 | 2019-10-08 16:10:43 |
|      4980 | default |        0 | payment/paypal_express_bml/productpage_display   | 0       | 2019-10-15 16:09:25 |
|      4981 | default |        0 | payment/paypal_express_bml/productpage_position  | 0       | 2019-10-08 16:10:43 |
|      4982 | default |        0 | payment/paypal_express_bml/productpage_size      | 190x100 | 2019-10-08 16:10:43 |
|      4983 | default |        0 | payment/paypal_express_bml/checkout_display      | 0       | 2019-10-15 16:09:25 |
|      4984 | default |        0 | payment/paypal_express_bml/checkout_position     | 0       | 2019-10-08 16:10:43 |
|      4985 | default |        0 | payment/paypal_express_bml/checkout_size         | 234x60  | 2019-10-08 16:10:43 |
|      4986 | default |        0 | payment/payflow_express_bml/active               | 0       | 2019-10-08 16:10:43 |
|      5595 | default |        0 | payment/wps_express_bml/active                   | 0       | 2019-10-08 16:10:43 |
+-----------+---------+----------+--------------------------------------------------+---------+---------------------+
16 rows in set (0.01 sec)
```

This is possibly related to #25077

**Comment by ajijshekh123** [comment]
Give me @magento give me 2.3-develop instance

**Comment by magento-engcom-team** [comment]
Hi @ajijshekh123. Thank you for your request. I'm working on Magento 2.3-develop instance for you

**Comment by magento-engcom-team** [comment]
Hi @ajijshekh123, here is your [Magento instance](https://i-25078-2-3-develop.instances.magento-community.engineering).
Admin access: [https://i-25078-2-3-develop.instances.magento-community.engineering/admin](https://i-25078-2-3-develop.instances.magento-community.engineering/admin)
Login: `admin` Password: `123123q`
Instance will be terminated in up to 3 hours.

**Comment by ajijshekh123** [comment]
I have reviewed this issue. I haven't entered the Publish ID on 'Paypal Credit' options. In the Shopping cart page there are 2 buttons are displaying the 'Paypal' & 'Paypal Credit' button.
I think it will Paypal features or It may be an issue.

See attached screenshot:

![Paypal Payment_Screenshot](https://user-images.githubusercontent.com/24711761/66888977-4d5a3580-effe-11e9-8e0c-9dbd61de5662.png)

![Shopping Cart_Paypal](https://user-images.githubusercontent.com/24711761/66888978-4d5a3580-effe-11e9-8eda-1936f9aa0bda.png)

Thanks.

**Comment by magento-engcom-team** [comment]
:white_check_mark: Confirmed by @ajijshekh123 
 Thank you for verifying the issue. Based on the provided information internal tickets `MC-21856` were created

**Issue Available**: @ajijshekh123, _You will be automatically unassigned. Contributors/Maintainers can claim this issue to continue. To reclaim and continue work, reassign the ticket to yourself._

**Comment by ky2vell** [comment]
Inside of:
/vendor/magento/module-paypal/Model/AbstractConfig.php
replace:
'payment/paypal_express/disable_funding_options'
with:
'paypal/style/disable_funding_options'

**Comment by devamitbera** [comment]
@sudheers-kensium where the solution? You have referred #25077.

it is confusion.

Please refer the right solution.

**Comment by engcom-Bravo** [comment]
The issue is already fixed in the latest 2.4-develop. No _PayPal Credit_ button in MiniCart, Checkout, Select Payment Method
![paypal](https://user-images.githubusercontent.com/51680745/106259743-4a517b80-6228-11eb-9435-7135bf2b2e53.png)

We are closing this issue.

**Comment by magento-engcom-team** [comment]
Hi @MichaelThessel, @engcom-Bravo. 

Thank you for your report and collaboration!

The related internal Jira ticket `MC-24937` was closed as `Fixed`. 

The fix will be available with the upcoming `2.4.3` release.
